packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -487,25 +492,27 @@ function formatAuthorTimestamp(username, timestamp) {
 	z-index: 2;
 	isolation: isolate;
 }
+
 .tab-group {
 	z-index: 2;
 	isolation: isolate;
 	position: relative;
 }
 
-section {
+.p-splitter {
 	display: flex;
-	flex-direction: column;
 	flex: 1;
-	overflow-x: auto;
-	overflow-y: hidden;
+	background: none;
+	border: none;
 }
 
-.p-splitter {
+section,
+.p-splitter:deep(.project-page) {
```

Comment:
Why do we need the `project-page` class if all Splitters have it?

Reply:
Because :deep effects the children components regardless and `tera-model.vue` is one of them and `tera-model.vue` has a splitter :(